scale_build/packages/build.py

```diff
@@ -160,7 +171,10 @@ def build_command(self):
             return self.buildcmd
         else:
             build_env = f'DEB_BUILD_OPTIONS={self.deoptions} ' if self.deoptions else ''
-            return [f'{build_env} debuild {" ".join(self.deflags)}']
+            env_flags = [
+                f'-e{k}={shlex.quote(v)}' for k, v in self._get_chroot_env().items()
```

Comment:
How secret are these secrets? Is it a problem that these will be visible in `ps ax` output? Or does debuildhide them?

Reply:
It shows up like this now
```
Running build command: ' debuild -eRELEASE_VERSION -eSENTRY_AUTH_TOKEN --no-lintian -j32 -us -uc -b'
```